pkg/arvo/app/publish.hoon

```diff
@@ -542,6 +547,59 @@
 ::
 |_  bol=bowl:gall
 ::
+++  get-last-update
+  |=  [host=@p book-name=@tas]
+  ^-  (unit @da)
+  =/  book  (~(get by books) host book-name)
+  ?~  book  ~
+  =/  wen  date-created.u.book
+  %-  some
+  %-  ~(rep by notes.u.book)
+  |=  [[@tas =note] out=_wen]
+  ^-  @da
+  %+  max  out
+  %+  max  last-edit.note
+  %-  ~(rep by comments.note)
+  |=  [[@da =comment] out=_out]
+  (max date-created.comment out)
+::
+++  get-notebook-from-date
+  |=  [host=@p book-name=@tas wen=@da]
+  ^-  notebook
+  =/  book  (~(got by books) host book-name)
+  %=  book
+      notes
+    %-  ~(rep by notes.book)
+    |=  [[nom=@tas not=note] out=(map @tas note)]
+    ^-  (map @tas note)
+    ?:  (gth last-edit.not wen)
+      (~(put by out) nom not)
+    =.  comments.not
+      %-  ~(rep by comments.not)
+      |=  [[nam=@da com=comment] out=(map @da comment)]
+      ?:  (gth date-created.com wen)
+        (~(put by out) nam com)
+      out
+    ?~  comments.not
+      out
+    (~(put by out) nom not)
+  ==
+::
+++  merge-notebooks
+  |=  [base=notebook diff=notebook]
+  ^-  notebook
+  %=  diff
+      notes
```

Comment:
ftr we discussed out of band that we're fine (for now) with not propagating deletion in these cases. Deletion in general will be picked up whenever the "archival" work happens.

Might still be nice to have a `::NOTE` here explaining that this is deliberate.